components/hero-unit.jsx

```diff
@@ -7,11 +7,16 @@ var HeroUnit = React.createClass({
   render: function() {
     return (
```

Comment:
Idk if this will help, but I had a similar problem recently and kind of went about it in an overly complicated way: https://github.com/mozilla/teach.webmaker.org/blob/develop/components/imagetag.jsx#L4-L16 (mostly due to the fact that CSS couldn't solve my problem)